src/main/client/get_cdtctx_base64.c

```diff
@@ -80,7 +74,23 @@ PyObject *AerospikeClient_GetCDTCTXBase64(AerospikeClient *self, PyObject *args,
     as_static_pool static_pool;
     memset(&static_pool, 0, sizeof(static_pool));
 
-    if (get_cdt_ctx(self, &err, &ctx, py_cdtctx, &ctx_in_use, &static_pool,
+    // Convert Python cdt_ctx to C version
+    // Pass in ctx into a dict so we can use helper function
+    PyObject *op_dict = PyDict_New();
```

Comment:
I think this dictionary needs to be decrefed after it is done being used in order to prevent reference count leaks.

Reply:
it's done